packages/@expo/cli/src/start/doctor/typescript/TypeScriptProjectPrerequisite.ts

```diff
@@ -5,20 +5,29 @@ import path from 'path';
 import * as Log from '../../../log';
 import { fileExistsAsync } from '../../../utils/dir';
 import { env } from '../../../utils/env';
+import { memoize } from '../../../utils/fn';
 import { everyMatchAsync, wrapGlobWithTimeout } from '../../../utils/glob';
 import { ProjectPrerequisite } from '../Prerequisite';
 import { ensureDependenciesAsync } from '../dependencies/ensureDependenciesAsync';
 import { updateTSConfigAsync } from './updateTSConfig';
 
 const debug = require('debug')('expo:doctor:typescriptSupport') as typeof console.log;
 
+const warnDisabled = memoize(() => {
+  Log.warn('Skipping TypeScript setup: EXPO_NO_TYPESCRIPT_SETUP is enabled.');
```

Comment:
Is this being wrapped inside memoize only to warn once?

Reply:
yes